### DISCUSSION TITLE
TikTok issue

### DISCUSSION THREAD

**Discussion by HsiangHo** [PlayCover/PlayCover#628] [General]
I don't know why TikTok is judged as a malicious app. Does anyone know?
from PR(#450).
<img width="479" alt="image" src="https://user-images.githubusercontent.com/29297882/209561423-d1ef0dba-96c7-42d6-bf82-6f412a6468e4.png">

**Comment by ohaiibuzzle** [comment]
This thing, last time I ran it, did this with no question asked (along with a bunch of other stuff)
![image](https://user-images.githubusercontent.com/23693150/209562294-6b50e04d-da6e-49a9-bb3b-977b9f7e1f93.png)
(That's the least of its problems, btw. Just use the web version if you don't want your Home folder to potentially be absolutely nuked)

**Reply by ohaiibuzzle** [reply]
Another thing it does is that during its Clear Cache operation, it blindly wipes the entire app container
And if you know these, you know that some stuff like Downloads, Movies, Pictures, Music and Desktop are symlinked into the app container on macOS
And... yeah. We'd better keep our users' data, privacy, and Mac safe.
